basics/getting-started/troubleshooting-pinot.md

```diff
@@ -1,32 +1,35 @@
 # Troubleshooting Pinot
 
-## Is there any debug information available in Pinot?
+## Find debug information in Pinot
 
-Pinot offers various ways to assist with troubleshooting and debugging problems that might happen. It is recommended to start off with the debug api which may quickly surface some of the commonly occurring problems. The debug api provides information such as tableSize, ingestion status, any error messages related to state transition in server, among other things.
+Pinot offers various ways to assist with troubleshooting and debugging problems that might happen. 
 
-The table debug api can be invoked via the Swagger UI as follows:
+Start with the debug api which will surface many of the commonly occurring problems. The debug api provides information such as tableSize, ingestion status, and error messages related to state transition in server.
+
+The table debug api can be invoked via the Swagger UI, as in the following image:
 
 ![Swagger - Table Debug Api](<../../.gitbook/assets/image (11) (2).png>)
 
 It can also be invoked directly by accessing the URL as follows. The api requires the `tableName`, and can optionally take `tableType (offline|realtime)` and `verbosity` level.
 
-```
+```bash
 curl -X GET "http://localhost:9000/debug/tables/airlineStats?verbosity=0" -H "accept: application/json"
 ```
 
-Pinot also provides a wide-variety of operational metrics that can be used for creating dashboards, alerting and [monitoring](https://docs.pinot.apache.org/operators/operating-pinot/monitoring). Also, all pinot components log debug information related to error conditions that can be used for troubleshooting.
+Pinot also provides a variety of operational metrics that can be used for creating dashboards, alerting and [monitoring](https://docs.pinot.apache.org/operators/operating-pinot/monitoring). 
+
+Finally, all pinot components log debug information related to error conditions.
 
-## How do I debug a slow query or a query which keeps timing out
+## Debug a slow query or a query which keeps timing out
 
-Please use these steps:
+Use the following steps:
 
 1. If the query executes, look at the query result. Specifically look at `numEntriesScannedInFilter` and `numDocsScanned`.
```

Comment:
I should look to see if these are defined in detail somewhere, and if so, link from here to that info.